latex-gitCheatSheet/FSFW_git-Spickzettel.tex

```diff
@@ -94,162 +97,165 @@
 		\sourcetext{color.ui auto}
-        \subsection{Improve interactive user-experience}
+        \subsection{Verbesserung der interaktiven Nutzererfahrung}
                 \sourcetext{interactive.singlekey true}
 }
 
 %% Create git Repositories
 \colouredbox{magenta}{
-	\begin{center}\section{Creating or cloning repositories}\end{center}
-	\subsection{From existing data}
+	\begin{center}\section{Erstellen und klonen von Repositorien}\end{center}
+	\subsection{Von existierenden Daten}
 		\cmdline{cd \ my\_project\_dir}\\
 		\cmdline{git init}\\
 		\cmdline{git add .}
                 \cmdline{git commit -m 'initial commit'}
-	\subsection{From existing repo}
+	\subsection{Von existierendem Repo}
 		\cmdline{git clone path/to/existing/repo path/to/new/repo}\\
 		\cmdline{git clone you@host.de:dir/project.git}\\
 		\cmdline{git clone http://[USER@]host.de/project.git}
 }
 
 %% Browse through files/repository
 \colouredbox{purple}{
-	\begin{center}\section{Getting information}\end{center}
-	\subsection{Files changed in working directory}
+	\begin{center}\section{Informationen erhalten}\end{center}
+	\subsection{Veränderte Dateien im Arbeitsverzeichnis}
 		\cmdline{git status}
-	\subsection{Changes of tracked files}
+	\subsection{Änderungen an überwachten Dateien}
 		\cmdline{git diff}
-	\subsection{Changes between ID1 and ID2}
+	\subsection{Änderungen zwischen ID1 und ID2}
 		\cmdline{git diff $<$ID1$>$ $<$ID2$>$}
-	\subsection{History of changes}
+	\subsection{Historie der Änderungen}
 		\cmdline{git log}
                 \cmdline{gitk}
 }
 
 %% Commit your changes
 \colouredbox{red}{
-	\begin{center}\section{Working with the Index and committing changes}\end{center}
-        \subsection{Add all changes in a file or directory to the index}
+	\begin{center}\section{Arbeiten mit dem Index und committen (dt.: einreichen) von Änderungen}\end{center}
+        \subsection{Füge alle Änderungen in einer Datei oder Verzeichnis zum Index hinzu}
                 \cmdline{git add path/to/add}
-        \subsection{Interactively select changes for addition/committing}
+        \subsection{Wähle interaktiv alle Änderungen zum Hinzufügen/ Committen aus}
                  \cmdline{git add -p [path/to/preselect/changes]}
                  \cmdline{git commit -p}
-        \subsection{Commit changes added to the index}
+        \subsection{Committe alle hinzugefügten Änderungen zum Index}
                 \cmdline{git commit}
-	\subsection{Add and commit all local changes}
+	\subsection{Füge alle lokalen Änderungen hinzu und committe sie}
 		\cmdline{git commit -a}
-	\subsection{Commit changes without editing the message}
+	\subsection{Committe Änderungen mit direkter Angabe einer Änderungsnachricht}
 		\cmdline{git commit -m "<message>"}
 }
 
 %% How to handle branches
 \colouredbox{orange}{
-	\begin{center}\section{Working with Branches}\end{center}
-	\subsection{List all branches}
+	\begin{center}\section{Arbeiten mit Branches (dt.: Äste)}\end{center}
```

Comment:
"Zweige" anstatt "Äste" vielleicht? Das ergibt mehr Sinn, weil "Zweig" auch in sowas wie "Abzweigung" drin steckt.

Reply:
+1 für Zweig. Wir haben das Wort Zweig auch als deutsches Synonym zu Branch im Workshop eingeführt.

Reply:
Die Argumentation finde ich nachvollziehbar auch wenn ich eigentlich "Zweig" mit "twig" übersetzen würde.